src/operator/controllers/intents_reconcilers/network_policy.go

```diff
@@ -117,6 +118,12 @@ func (r *NetworkPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	err = r.removeOrphanNetworkPolicies(ctx)
+	if err != nil {
+		r.RecordWarningEventf(intents, ReasonRemovingNetworkPolicyFailed, "could not remove orphan network policies: %s", err.Error())
```

Comment:
What do these network policies represent from the user's perspective? Are they network policies created by intents? Please change the event text to a message that the user could make sense of without context of the implementation (orphan network policies is an implementation detail)